README.md

```diff
@@ -39,7 +39,7 @@ repo.
 
 If you are new here, please read the getting started docs:
 
-* [logs](./docs/logs/getting-started/README.md)
+* logs: [Console](./docs/logs/getting-started-console/README.md)
```

Comment:
Not related to this PR but I think we should capitalize these words: `Logs`, `Metrics`, and `Traces`.

Reply:
Updated, thanks!